app/src/pages/experiment/ExperimentCompareTable.tsx

```diff
@@ -251,33 +251,31 @@ export function ExperimentCompareTable(props: ExampleCompareTableProps) {
       {
         header: "input",
         accessorKey: "input",
-        minWidth: 500,
         cell: ({ row }) => {
           return (
             <CellWithControlsWrap
               controls={
-                <TooltipTrigger>
-                  <Button
-                    size="S"
-                    aria-label="View example details"
-                    leadingVisual={<Icon svg={<Icons.ExpandOutline />} />}
-                    onPress={() => {
-                      startTransition(() => {
-                        setDialog(
-                          <Suspense>
-                            <ExampleDetailsDialog
-                              exampleId={row.original.example.id}
-                              onDismiss={() => {
-                                setDialog(null);
-                              }}
-                            />
-                          </Suspense>
-                        );
-                      });
-                    }}
-                  />
-                  <Tooltip>View Example</Tooltip>
-                </TooltipTrigger>
+                <Button
```

Comment:
removing the tooltips as I think it's pretty hard to find these buttons right now. Probably will do a design pass to make these always pinned